db/schema.ts

```diff
@@ -124,6 +124,18 @@ export const RoutineJobsTable = pgTable("routine_jobs", {
 
 export type RoutineJob = typeof RoutineJobsTable.$inferSelect;
 
+export const RoutineJobExecutionsTable = pgTable("routine_job_executions", {
```

Comment:
Delete this table, we don't need schema changes. Instead the vellum client sdk should have an endpoint we should use for listing workflow deployment event executions. Use it - look up the docs at docs.vellum.ai if need be